tests/eas/load_tracking.py

```diff
@@ -943,7 +943,7 @@ class _PELTTaskGroupsTest(LisaTest):
         'modules': ['cpufreq', 'cgroups'],
     }
     # Allowed error margin
-    allowed_util_margin = 0.02
+    allowed_util_margin = 0.04
```

Comment:
Why do we need to increase this... that's a 4% margin, right?
What's the failure rate if we keep it to 2%

Reply:
I've also just notice that `allowed_util_margin` this seems to be something similar to `ERROR_MARGIN_PCT` defined at the top of the file... where it's defined to be 15%

Despite I reviewed @Elieva code, I did not notice this difference before... which could explain why the original tests from @mdigiorgio was passing and these new are not.

Should not we use the same margin across all these tests?... and in that case I would really like the smallest number (2% or 4%)... I'm just concerned that above tests could fails... but maybe it's for the same reason this patch is trying to fix... and then perhaps we can find a solution suitable for both casese.

Reply:
I don't remember the exact figure, but with 4%, we don't have any failure at all on 800 iterations. I haven't reviewed all the tests to see how they use the margin. It would make little sense to me to force using the same margin for unrelated quantities. The 15% may benefit from more testing to see if it actually needs to be that large.

Reply:
Well, it's seems to be used also for PELT realted tests, in `PELTTasksTest`... but, in any case, maybe we can keep this bit for another time.